webpack.dev.js

```diff
@@ -61,6 +61,6 @@ module.exports = merge.smart(require('./webpack.base.js'), {
     historyApiFallback: true,
     disableHostCheck: true,
     headers: {"Access-Control-Allow-Origin": "*"},
-    setup: fileSystemAPIPlugin,
+    before: fileSystemAPIPlugin,
```

Comment:
@tech4him1 good catch, I ran this for the last 4 days and did not notice the warning 👍